zulip_bots/zulip_bots/bots/tictactoe/doc.md

```diff
@@ -1,28 +1,30 @@
-# About Tic-Tac-Toe Bot
+# Tic-Tac-Toe Bot
 
 This bot allows you to play tic-tac-toe in a private message with the bot.
 Multiple games can simultaneously be played by different users, each playing
 against the computer.
 
 The bot only responds to messages starting with @mention of the bot(botname).
 
```

Comment:
Can you add a section `Setup` here as well? If the bot doesn't require any special setup, just add smt like "This bot doesn't require any special setup. Just run it according to the instructions in \<link-to-running-bots-guide\>".